File: P161990.py
def solution(wallpaper):
    l, t = 50, 50
    r, b = 0, 0
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == "#":
                if i < t:
                    t = i
                if i > b:
                    b = i
                if j < l:
                    l = j
                if j > r:
                    r = j
    return [t, l, b+1, r+1]

File: P161990.java
public class P161990 {
    public int[] solution(String[] wallpaper) {
        int l = 50;
        int t = 50;
        int r = 0;
        int b = 0;
        
        for (int i=0; i<wallpaper.length; i++) {
            for (int j=0; j<wallpaper[0].length(); j++) {
                if (wallpaper[i].charAt(j) == '#') {
                    if (i < t) t = i;
                    if (i > b) b = i;
                    if (j < l) l = j;
                    if (j > r) r= j;
                }
            }
        }
        return new int[] {t, l, b+1, r+1};
    }
}
